templates/views/single-p4-form.twig

```diff
@@ -1,21 +1,25 @@
-<form id="p4en_form" action="{{ form_submit_url }}" method="post" class="p4fb_form p4en_form">
+{% if (form_errors) %}
+	<div class="errors"><p>There were errors. Please check your answer.</p></div>
+{% endif %}
```

Comment:
I think the error message would be better inside the form tag for accessibility.